examples/cart/index.js

```diff
@@ -52,7 +52,9 @@ $(function() {
 
     attachCheckoutButtonListeners();
 
-    closeCart();
+    closeCartHandler();
+    overlayHandler();
+    keyPressHandler();
```

Comment:
Any reason to keep these separated out as their own functions? Seems like it might be a little more simple to have them squashed into a single `bindEventListeners()` or similar call, particularly given the number of them on this page.